### PR TITLE
Recreated last updated date functionality

### PR DESCRIPTION
**Pull request by AdamZarger**
Adding logic to display a "last updated" date as a subnote of the title. This date is pulled from the Elasticsearch `_meta` index. In the case of an error: the "last updated" message is not displayed.

## Additions

- "last updated" date added to UI

## Review

- @JeffreyMFarley 
- @sephcoster 

## Todos

- Write tests for the SearchPanel

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/__tests__/__snapshots__/SearchPanel.spec.jsx.snap [hunk 1]
```diff
@@ -0,0 +1,123 @@
+exports[`SearchPanel renders renders with last updated date 1`] = `
+ShallowWrapper {
```
Comment (JeffreyMFarley) [inline src/__tests__/__snapshots__/SearchPanel.spec.jsx.snap]: This is not what a snapshot looks like. It should look like regular HTML

File: src/__tests__/SearchPanel.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  return {
+    target,
+    props
+  }
+}
+
+describe('SearchPanel', () => {
+  let target, props
+  beforeEach(() => {
+    ({ target, props } = setupEnzyme())
```
Comment (JeffreyMFarley) [inline src/__tests__/SearchPanel.spec.jsx]: For snapshots, you will want to emulate code like `setupSnapshot` in the codebase, not `setupEnzyme`.  Enzyme is used for testing events, like 'click mouse' or 'keydown', not rendering

File: src/__tests__/SearchPanel.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { SearchPanel } from '../SearchPanel';
+import React from 'react'
+import renderer from 'react-test-renderer'
+import thunk from 'redux-thunk'
+import { shallow } from 'enzyme';
+
+function setupEnzyme(initialProps={}) {
+  const props = Object.assign({
+    lastUpdated: new Date( '2016-02-01T05:00:00.000Z' )
```
Comment (JeffreyMFarley) [inline src/__tests__/SearchPanel.spec.jsx]: This is nicely timezone independent

File: src/SearchPanel.jsx [hunk 4]
```diff
@@ -1,14 +1,31 @@
...
-      <h2>Search complaint data</h2>
-      <SearchBar />
-      <PillPanel />
-    </div>
+export class SearchPanel extends React.Component {
+  render() {
+    var lastUpdatedMessage = null;
 
+    if ( this.props.lastUpdated ) {
+      lastUpdatedMessage = <span className="date-subscript">(last updated: <FormattedDate value={ this.props.lastUpdated } />)</span>
```
Comment (JeffreyMFarley) [inline src/SearchPanel.jsx]: This line might be dinged by the linter as too long

Comment (AdamZarger) [reply src/SearchPanel.jsx]: I've fixed this issue in the latest commit, thanks for pointing it out!

File: src/__tests__/__snapshots__/SearchPanel.spec.jsx.snap [hunk 5]
```diff
@@ -0,0 +1,167 @@
+exports[`component:SearchPanel displays last updated date when present 1`] = `
+<div
+  className="search-panel">
+  <h2>
+    Search complaint data
```
Comment (JeffreyMFarley) [inline src/__tests__/__snapshots__/SearchPanel.spec.jsx.snap]: ...but no date is displayed here.  That's why the coverage is not at 100%

https://coveralls.io/builds/13474174/source?filename=src%2FSearchPanel.jsx

For some reason, in your test setup, `results.lastUpdated` is not being set like you think it is

**Pull request merged**